qemu_test.go

```diff
@@ -395,10 +395,16 @@ func TestQemuSetMemoryResources(t *testing.T) {
 
 	q := &qemu{}
 
+	hostMem, err := getHostMemorySize()
+	if err != nil {
+		t.Fatal(err)
+	}
+	memMax := fmt.Sprintf("%dM", int(float64(hostMem)/1024/1024)+1024)
```

Comment:
I'd add a paranoid check to ensure that `hostMemBytes > 0` before the division. It would also be clearer potentially to define variables for the division + addition with comments explaining why those values were chosen.